rpc/rpc.c

```diff
@@ -286,24 +286,22 @@ M0_INTERNAL m0_bcount_t m0_rpc_max_seg_size(struct m0_net_domain *ndom)
 {
 	M0_PRE(ndom != NULL);
 
-#ifdef ENABLE_LUSTRE
-	return min64u(m0_net_domain_get_max_buffer_segment_size(ndom),
+	if (m0_streq(ndom->nd_xprt->nx_name, "lnet"))
```

Comment:
This should be ndom->nd_xprt == &m0_net_lnet_xprt. The same below.

Reply:
incorporated